store/connections/remote-desktop.yml

```diff
@@ -83,25 +83,40 @@ additionalInformation: >-
   </Note>
 
 
+  <Note>
+
+
+  Ensure that the Hoop Agent you are connecting to has the HOOP_GATEWAY_URL environment variable set to wss://hostname:port.
+
+
+  </Note>
+
+
   <Steps>
     <Step title="Download and Install the RDP Manager Client">
-      [Remote Desktop Manager Client](https://devolutions.net/remote-desktop-manager/download/)
+      \\[Remote Desktop Manager Client](https://devolutions.net/remote-desktop-manager/download/)
```

Comment:
hey @nsaronson double check your local decap admin to remove these double slashes from the document, then it would be good to go